packages/calypso-e2e/src/element-helper.ts

```diff
@@ -1,10 +1,16 @@
 import { Locator, Page } from 'playwright';
 import envVariables from './env-variables';
 
+const navTabParent = 'div.section-nav';
+
 const selectors = {
+	// General
+	main: 'main[role="main"]',
+
 	// clickNavTab
-	navTab: ( tab: string ) => `.section-nav-tab:has-text("${ tab }")`,
-	mobileNavTabsToggle: 'button.section-nav__mobile-header',
+	navTabItem: ( { name = '', selected }: { name?: string; selected?: boolean } = {} ) =>
+		`${ navTabParent } a[aria-current="${ selected }"]:has(span:has-text("${ name }"))`,
```

Comment:
I've made this selector slightly more complex but this allows us to specify any four permutation of the input options and thus allowing us to avoid having to embed a selector within template literals (which we do in a few other places):

```typescript
`${ selectors.navTabItem( 'navtabname' ) }[aria-current="selected"]`
```